202200058/10505_소득_불균형.py

```diff
@@ -0,0 +1,8 @@
+for k in range(int(input())):
+    n=int(input())
+    people=list(map(int,input().split()))
+    total=0
+    for i in people:
+        if i <= sum(people)/n:
+            total+=1
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    avg = sum(people) / len(people)
    for i in people:
        if avg >= i:
            total += 1
```
평균 값을 `for`문 밖에서 한 번만 계산하는 방법도 있습니다 😄